README.md

```diff
@@ -1,2 +1,14 @@
 # CVScanAI
 AI Driven CV Validator
+
+## Installation
+
+1. ```pip install -r cv_scan_ai/requirements.txt```
+2. ```export OPENAI_API_KEY="sk-...."```
+
```

Comment:
Would be useful to specify which version of python is expected and run it in venv
```
python3.9 -m venv .venv_cv_scan_ai
source .venv_cv_scan_ai/bin/activate
```